plugin/pkg/auth/authorizer/node/graph.go

```diff
@@ -334,6 +335,10 @@ func (g *Graph) recomputeDestinationIndex_locked(n graph.Node) {
 //   pvc       -> pod
 //   svcacct   -> pod
 func (g *Graph) AddPod(pod *corev1.Pod) {
+	start := time.Now()
+	defer func() {
+		graphActionsDuration.WithLabelValues("AddPod").Observe(time.Since(start).Seconds())
+	}()
```

Comment:
Why is DeletePod not intstrumented?

Reply:
Yes, good point. I added metrics to DeletePod and SetNodeConfigMap.